### PR TITLE
Fixes #16605: Beautify and improve UX/UI

### PR DESCRIPTION
**Pull request by ElaadF**
https://issues.rudder.io/issues/16605

### CODE REVIEW

File: user-management/src/main/elm/sources/DataTypes.elm [hunk 1]
```diff
@@ -83,10 +82,9 @@ type Msg
     | Login String
     | AddRole User String
     | RemoveRole User String
-    | SubmitUpdatedInfos User
-    | SubmitNewUser User Password
+    | SubmitUpdatedInfos User (List String)
```
Comment (VinceMacBuche) [inline user-management/src/main/elm/sources/DataTypes.elm]: I think the list of role you want to save must be a part of the user you are updating

**Comment by ElaadF** [comment]
PR updated with a new commit

**Comment by ElaadF** [comment]
PR updated with a new commit

**Comment by ElaadF** [comment]
Commit modified

**Pull request merged**